coderd/coderd.go

```diff
@@ -280,7 +280,7 @@ func New(options *Options) *API {
 				// file content is expensive so it should be small.
 				httpmw.RateLimitPerMinute(12),
 			)
-			r.Get("/{hash}", api.fileByHash)
+			r.Get("/{hashID}", api.fileByID)
```

Comment:
Remove the word hash?
```suggestion
			r.Get("/{fileID}", api.fileByID)
```